geomstats/geometry/quotient_manifold.py

```diff
@@ -0,0 +1,480 @@
+"""Classes for fiber bundles and quotient metrics."""
+
```

Comment:
How about having 2 files: quotient.py and fiber_bundle.py?

For users who are browsing through the geomstats' geometry (e.g. directory `geometry`), they would more directly see that fiber bundle is available. Otherwise, it is a bit hidden.

Reply:
True

Reply:
Should I split the tests in two files as well? There would be redunduncies

Reply:
Yes, split into two would be good. For the redundant tests, maybe just put it in one of the tests? (if it is testing the same code anyway, and would not increase the code coverage)?